auction/run.sh

```diff
@@ -6,26 +6,28 @@ then
     exit
 fi
 
-# The private key and address of the first bidder.
-# Swap these into program.json, when running transactions as the first bidder.
-# "private_key": "APrivateKey1zkpG9Af9z5Ha4ejVyMCqVFXRKknSm8L1ELEwcc4htk9YhVK"
-# "address": aleo1yzlta2q5h8t0fqe0v6dyh9mtv4aggd53fgzr068jvplqhvqsnvzq7pj2ke
+echo "
+We will be playing the role of three parties.
 
-# The private key and address of the second bidder.
-# Swap these into program.json, when running transactions as the second bidder.
-# "private_key": "APrivateKey1zkpAFshdsj2EqQzXh5zHceDapFWVCwR6wMCJFfkLYRKupug"
-# "address": aleo1esqchvevwn7n5p84e735w4dtwt2hdtu4dpguwgwy94tsxm2p7qpqmlrta4
+The private key and address of the first bidder.
+private_key: APrivateKey1zkpG9Af9z5Ha4ejVyMCqVFXRKknSm8L1ELEwcc4htk9YhVK
+address: aleo1yzlta2q5h8t0fqe0v6dyh9mtv4aggd53fgzr068jvplqhvqsnvzq7pj2ke
 
-# The private key and address of the auctioneer.
-# Swap these into program.json, when running transactions as the auctioneer.
-# "private_key": "APrivateKey1zkp5wvamYgK3WCAdpBQxZqQX8XnuN2u11Y6QprZTriVwZVc",
-# "address": "aleo1fxs9s0w97lmkwlcmgn0z3nuxufdee5yck9wqrs0umevp7qs0sg9q5xxxzh"
+The private key and address of the second bidder.
+private_key: APrivateKey1zkpAFshdsj2EqQzXh5zHceDapFWVCwR6wMCJFfkLYRKupug
+address: aleo1esqchvevwn7n5p84e735w4dtwt2hdtu4dpguwgwy94tsxm2p7qpqmlrta4
 
+The private key and address of the auctioneer.
+private_key: APrivateKey1zkp5wvamYgK3WCAdpBQxZqQX8XnuN2u11Y6QprZTriVwZVc
+address: aleo1fxs9s0w97lmkwlcmgn0z3nuxufdee5yck9wqrs0umevp7qs0sg9q5xxxzh
+"
 
 echo "
+Let's start an auction!
+
 ###############################################################################
 ########                                                               ########
-########            STEP 0: Initialize a new 2-party auction           ########
+########               Initialize a new 2-party auction                ########
```

Comment:
Why remove the steps here?
I think it is additive to the dev experience to follow along with the steps in the script and then try with theirs locally.